src_py/freetype.py

```diff
@@ -55,6 +57,15 @@ def __init__(self, file, *args, **kwargs):
             if isinstance(file, compat.unicode_):
                 # ...but we can't pass a unicode path to Freetype,
                 # so we'll open the file in Python and pass the handle instead.
-                file = open(file, 'rb')
```

Comment:
It may be possible to remove this code for Windows altogether now. _freetype should work with Unicode paths as of 864888f1d7f2.